src/views/wallet/UserAssetsScreen.tsx

```diff
@@ -1,61 +1,60 @@
 import React, { useMemo, useCallback, useRef } from 'react'
 
-import { SyncOutlined } from '@ant-design/icons'
 import * as RD from '@devexperts/remote-data-ts'
 import { Balances, Balance } from '@thorchain/asgardex-binance'
 import { getAssetFromString, assetAmount, bnOrZero, formatAssetAmountCurrency } from '@thorchain/asgardex-util'
-import { Row, Col, Table } from 'antd'
+import { Row, Col } from 'antd'
 import { ColumnType } from 'antd/lib/table'
 import { Option, some, none } from 'fp-ts/lib/Option'
 import * as O from 'fp-ts/lib/Option'
 import { useObservableState } from 'observable-hooks'
 import { useHistory } from 'react-router-dom'
 
 import ErrorView from '../../components/shared/error/ErrorView'
-import DynamicCoin from '../../components/shared/icons/DynamicCoin'
-import Button from '../../components/uielements/button'
+import Coin from '../../components/uielements/coins/coin'
 import Label from '../../components/uielements/label'
 import { useBinanceContext } from '../../contexts/BinanceContext'
 import * as walletRoutes from '../../routes/wallet'
+import { TableWrapper } from './UserAssetsScreen.style'
 
 const UserAssetsScreen: React.FC = (): JSX.Element => {
   const history = useHistory()
 
-  const { balancesState$, reloadBalances } = useBinanceContext()
+  const { balancesState$ } = useBinanceContext()
   const balancesRD = useObservableState(balancesState$, RD.initial)
 
   // store previous data of balances to still render these while reloading new data
   const previousBalances = useRef<Option<Balances>>(none)
 
-  const clickReloadHandler = () => {
-    reloadBalances()
-  }
-
   const iconColumn: ColumnType<Balance> = {
     key: 'icon',
     // TODO(@Veado): i18n
-    title: 'icon',
+    title: '',
+    width: 1,
     dataIndex: 'symbol',
-    render: (_, { symbol }) => <DynamicCoin type={symbol} size={'normal'} />
+    render: (_, { symbol }) => <Coin type={symbol} size="big" />
```

Comment:
Are logos used by `Coin` based on `Trust Wallet` repo (see https://github.com/thorchain/asgardex-electron/issues/37) ? If not, please implement this feature with another PR.